src/distributed_ls/pilut/DistributedMatrixPilutSolver.h

```diff
@@ -9,7 +9,7 @@
 #define _DISTRIBUTED_MATRIX_PILUT_SOLVER_HEADER
 
 #include "HYPRE_config.h"
-#include "hypre_general.h"
+#include "general.h"
```

Comment:
You should be able to just delete this include.  It looks like we have some a little bit of legacy header misuse.